docs-content/getting-started/fullstack-frameworks/next-js.md

```diff
@@ -274,6 +274,12 @@ export default function RootLayout({ children }: { children: React.ReactNode })
 }
 ```
 
+## Configure `inlineImages`
+
+We use a package called [rrweb](https://www.rrweb.io/) to record web sessions. rrweb enables inlining images into session replay to improve performance; however, the inlined images can cause CORS issues in some situations.
```

Comment:
> rrweb enables inlining images into session replay to improve performance

should be 

> rrweb supports inlining images into sessions to improve replay accuracy, so that images that are only available from your local network can be saved; however, ...